src/components/linter.ts

```diff
@@ -129,7 +130,24 @@ export class Linter {
                 return
             }
         }
-        this.extension.linterLogParser.parse(stdout)
+        const tabSize = this.getChktexrcTabSize()
+        this.extension.linterLogParser.parse(stdout, undefined, tabSize)
+    }
+
+    private getChktexrcTabSize(): number | undefined {
+        const filePath = this.rcPath
+        if (!filePath) {
+            this.extension.logger.addLogMessage('The .chktexrc file not found.')
+            return
+        }
+        const rcFile = fs.readFileSync(filePath).toString()
+        const reg = /^\s*TabSize\s*=\s*(\d+)\s*$/m
+        const match = reg.exec(rcFile)
+        if (match) {
+            return parseInt(match[1])
```

Comment:
Maybe, we should enforce base `10` here. The default value for the `radix` argument of `parseInt` seems to depend on the platform.

Reply:
I have chosen to use `Number`.

See https://stackoverflow.com/questions/4090518/what-is-the-difference-between-parseint-and-number